### DISCUSSION TITLE
Update File für DTU Fusion

### DISCUSSION THREAD

**Discussion by twitnic** [tbnobody/OpenDTU#2256] [Q&A] [answered]
![Bildschirmfoto 2024-09-05 um 15 17 54](https://github.com/user-attachments/assets/7e14d2db-6af9-4c6e-822a-ac128e4a1367)
Hallo :)

Ich habe 2 von diesen DTU "zu betreuen". Könnt ihr mir sagen, welche Datei ich zum Update nutzen kann?
Es ist ja schon ein ESP32-S3.

Danke euch.

**Comment by ms49434** [comment, marked as answer]
Für das Fusion Board nimmst du die opendtu-generic_esp32s3_usb.bin.

**Comment by Gururom** [comment]
Die USB Version doch nur wenn per Kabel am Rechner geupdatet wird, oder?

**Reply by mroenne2022** [reply]
Nein, genau die genannte Version kannst du natürlich zum Update über OTA nutzen, vollkommen okay. Hab ich selbst schon mehrfach durchgeführt. 
Am Rechner per Kabel zum erstmals Flash muss es die entsprechende factory sein. 
Auf dieser Seite kannst du es auch nochmal nachlesen:
https://github.com/markusdd/OpenDTUFusionDocs/blob/main/pics/ReleaseImages.png
